hashintel/hash
# Update entity version on entity editor

## 🌟 What is the purpose of this PR?

This PR adds version indicators to the entity type cards on entity editor, ands lets user update their entities to the newer versions. 

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203363157432094/1203515054881271/f) _(internal)_
- [Design](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-App?node-id=7049%3A253128&t=lUIA4jllwiLEIyBZ-4) _(internal)_

## 🔍 What does this change?

- Add `sx` prop to `WhiteCard`
- Fix a minor timestamp related issue

## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Create an entity type
2. Create an entity from that entity type
3. Go back and update the entity type by changing anything and publishing the changes.
1. Go back to the same entity you created at step 2
2. Confirm that entity type card has "update entity" button as shown as the video below, and it works as expected.

## 📹 Demo


https://user-images.githubusercontent.com/39553853/211270480-31b1b49d-49f9-49fc-8dc9-e82f49d44b2c.mov
